worker/lease/dead_manager_test.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package lease_test
+
+import (
+	"reflect"
+
+	gc "gopkg.in/check.v1"
+
+	lease "github.com/juju/juju/worker/lease"
+)
+
+type DeadManagerSuite struct {
+}
+
+var _ = gc.Suite(&DeadManagerSuite{})
+
+type DeadManagerError struct{}
+
+const DeadManagerErrorMessage = "lease manager stopped"
+
+func (*DeadManagerError) Error() string {
+	return DeadManagerErrorMessage
+}
+
+// This creates and new DeadManager and calls all of its exported methods with zero
+// values. All methods should return the specified DeadManagerError.
```

Comment:
This isn't true. It either returns the `DeadManagerError`, that is, the error passed to the `NewDeadManager` method; or it returns an error with the message "lease manager stopped". Here, in this tests, both are the same for the purpose of making this test pass. Once I find out what the desired behavior is, then I will update this test accordingly.